libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/ActionScope.cs

```diff
@@ -202,7 +202,6 @@ protected virtual async Task<DialogTurnResult> BeginActionAsync(DialogContext dc
             {
                 { "DialogId", action.Id },
```

Comment:
> action.Id  [](start = 29, length = 11)

this should be "this.Id"

Reply:
Is it worth to leave action.Id as ActionId?